pybigtools/src/lib.rs

```diff
@@ -1453,13 +1459,14 @@ impl BBIRead {
                 let iter = Box::new(bigwig_average_over_bed(bedin, b, name));
                 BigWigAverageOverBedEntriesIterator { iter, usename }.into_py(py)
             }
-            BBIReadRaw::BigBedFile(_) | BBIReadRaw::BigBedFileLike(_) => return Err(BBIFileClosed::new_err("Not a bigWig.")),
+            BBIReadRaw::BigBedFile(_) | BBIReadRaw::BigBedFileLike(_) => {
+                return Err(BBIFileClosed::new_err("Not a bigWig."))
+            }
             #[cfg(feature = "remote")]
             BBIReadRaw::BigBedRemote(_) => return Err(BBIFileClosed::new_err("Not a bigWig.")),
```

Comment:
Are these error types / messages correct?

Reply:
Ah that doesn't look right. Not sure the right exception but it shouldn't be file closed.

Thinking about it, could make BigWig and BigBed subclasses which effectively have no differences except extra methods. 

Or just also implement this for bigBeds akin to values.